src/commands/typescript/update.ts

```diff
@@ -55,7 +55,7 @@ export default class Update extends SfdxCommand {
     this.ux.warn('This is for testing new versions only. To update the version you must go through dev-scripts.');
 
     this.updateTsVersion();
-    await this.updateEsTarget();
+    await Promise.all(this.packages.map((pkg) => this.updateEsTargetConfig(pkg.location)));
```

Comment:
turned the method into a parallel op

Reply:
`this.packages` used to be an array of lerna packages, I can't remember if there was a reason I didn't refactor these methods:
https://github.com/salesforcecli/plugin-release-management/blob/main/src/commands/typescript/update.ts#L70

this can be reverted back to the old code as it is always expected to run `typescript:update` in a single repo.

Reply:
I think PDT was still using it maybe?  I'll definitely switch it back.